patch/guides (#270)
nodejs/i18n: guides/TEAMS.md

# Teams

> Our localization process structure based on language community, so we have l10n team for each target locale. Read guide below to understand goals of these teams and discover how to find and join your own.

Reviewer: ```suggestion
Our localization process structure is based on language community, so we have one l10n team for each target locale. Read the guide below to understand the goals of these teams and discover how to find and join your own.
```

| Swahili    | [nodejs/nodejs-sw](https://github.com/nodejs/nodejs-sw)       | [@nodejs/nodejs-sw](https://github.com/orgs/nodejs/teams/nodejs-sw) |
| Swedish    | [nodejs/nodejs-sv](https://github.com/nodejs/nodejs-sv)       | none                                                                |
| Taiwan     | [nodejs/nodejs-zh-TW](https://github.com/nodejs/nodejs-zh-TW) | [@nodejs/nodejs-tw](https://github.com/orgs/nodejs/teams/nodejs-tw) |
| Tamil      | [nodejs/nodejs-ta](https://github.com/nodejs/nodejs-ta)       | [@nodejs/nodejs-ta](https://github.com/orgs/nodejs/teams/nodejs-ta) |
| Turkish    | [nodejs/nodejs-tr](https://github.com/nodejs/nodejs-tr)       | [@nodejs/nodejs-tr](https://github.com/orgs/nodejs/teams/nodejs-tr) |
| Ukrainian  | [nodejs/nodejs-uk](https://github.com/nodejs/nodejs-uk)       | [@nodejs/nodejs-uk](https://github.com/orgs/nodejs/teams/nodejs-uk) |
| Vietnamese | [nodejs/nodejs-vi](https://github.com/nodejs/nodejs-vi)       | [@nodejs/nodejs-vi](https://github.com/orgs/nodejs/teams/nodejs-vi) |

## About

> All these teams created just to simplify communication and management processes for each locale group. There is no any other functional reasons have this teams, that why we're not forcing you to join everywhere.

Reviewer: ```suggestion
All these teams exist to simplify communication and management processes for each locale group. There is no other functional reason these teams; you're not required to join them to participate.
```

### Repository

> GitHub repository for locale group is good place to host some language specific documentation, guidelines in markdown. You can collaborate to work on docs like this or just read. In other hand repository issues it's a great place to discuss some questions related to translations like glossary, style guide, and team management on both GitHub and Crowdin.

Reviewer: ```suggestion
The repositories for your locale group is a good place to host some language-specific documentation and guidelines in markdown files. You can collaborate to work on docs like this or just read. On the other hand, repository issues are a great place to discuss some questions related to translations like glossary, style guide, and team management on both GitHub and Crowdin.
```
